theatre/studio/src/panels/DetailPanel/propEditors/utils/useEditingToolsForPrimitivePropInDetailsPanel.tsx

```diff
@@ -17,49 +17,56 @@ import type {PropTypeConfig} from '@theatre/core/propTypes'
 import {isPropConfSequencable} from '@theatre/shared/propTypes/utils'
 import type {SequenceTrackId} from '@theatre/shared/utils/ids'
 
-interface CommonStuff<T> {
+interface EditingToolsCommon<T> {
   value: T
   beingScrubbed: boolean
   contextMenuItems: Array<IContextMenuItem>
+  /** e.g. `< • >` or `<   >` for {@link Sequenced} */
   controlIndicators: React.ReactElement
 
   temporarilySetValue(v: T): void
   discardTemporaryValue(): void
   permanentlySetValue(v: T): void
 }
 
-interface Default<T> extends CommonStuff<T> {
+interface Default<T> extends EditingToolsCommon<T> {
```

Comment:
2/10 rename `Default` -> `EditingToolsDefault`. Same for static and sequenced below.